lib/shutterstock-ruby/configuration.rb

```diff
@@ -0,0 +1,13 @@
+module ShutterstockRuby
+  class Configuration
+    attr_accessor :access_token
```

Comment:
Is `attr_reader` enough for these 3?

Reply:
Yeah could be, I was thinking you might want to change them after creating but on second thoughts that probably wont need to happen

Reply:
`attr_accessor` is required for the backwards compatibility